tests/performance/local/future_overhead.cpp

```diff
@@ -442,9 +442,10 @@ void measure_function_futures_create_thread_hierarchical_placement(
     high_resolution_timer walltime;
     for (std::size_t t = 0; t < num_threads; ++t)
     {
-        auto const hint = hpx::threads::thread_schedule_hint(t);
+        auto const hint =
+            hpx::threads::thread_schedule_hint(static_cast<std::int16_t>(t));
         auto spawn_func = [&thread_func, sched, hint, t, count, num_threads,
-                              desc, stack_size]() {
+                              desc, stack_size, prio]() {
```

Comment:
Is this actual non-compliance in MSVC or allowed behaviour according to the standard?

Reply:
Non-compliance in MSVC requires that even `const` variables have to be captured. OTH, clang/gcc emits a warning if a `const` variable is captured. So this PR makes it non-const to begin with.